From: Kevin Hilman <khilman@deeprootsystems.com>
Date: Mon, 2 Mar 2009 21:57:31 +0000 (-0800)
Subject: OMAP3: PM: CPUfreq support for OMAP3EVM board
X-Git-Url: http://git.kernel.org/?p=linux%2Fkernel%2Fgit%2Fkhilman%2Flinux-omap-pm.git;a=commitdiff_plain;h=e10fda5f6106c6e0a559c3a4720ebff7a8bb1a43

OMAP3: PM: CPUfreq support for OMAP3EVM board

From: Koen Kooi <koen@beagleboard.org>

Uses the common OMAP3 OPP settings on OMAP3 EVM board.

Signed-off-by: Kevin Hilman <khilman@deeprootsystems.com>
---

diff --git a/arch/arm/mach-omap2/board-omap3evm.c b/arch/arm/mach-omap2/board-omap3evm.c
index 6fbbe95..072930a 100644
--- a/arch/arm/mach-omap2/board-omap3evm.c
+++ b/arch/arm/mach-omap2/board-omap3evm.c
@@ -36,14 +36,11 @@
 #include <mach/usb-ehci.h>
 #include <mach/common.h>
 #include <mach/mcspi.h>
-#include <mach/omap-pm.h>
-#include <mach/clock.h>
 
 #include "sdram-micron-mt46h32m32lf-6.h"
 #include "twl4030-generic-scripts.h"
 #include "mmc-twl4030.h"
-#include "pm.h"
-#include "omap3-opp.h"
+
 
 static struct resource omap3evm_smc911x_resources[] = {
 	[0] =	{
@@ -220,8 +217,7 @@ struct spi_board_info omap3evm_spi_board_info[] = {
 
 static void __init omap3_evm_init_irq(void)
 {
-	omap2_init_common_hw(mt46h32m32lf6_sdrc_params, omap3_mpu_rate_table,
-	                     omap3_dsp_rate_table, omap3_l3_rate_table);
+	omap2_init_common_hw(mt46h32m32lf6_sdrc_params, NULL, NULL, NULL);
 	omap_init_irq();
 	omap_gpio_init();
 	omap3evm_init_smc911x();
